Source/CoreGraphicsPolyfill.swift

```diff
@@ -0,0 +1,493 @@
+//
+//  CoreGraphicsPolyfills.swift
+//  SVGView
+//
+//  Created by khoi on 10/5/25.
+//
+
+import Foundation
+
+public typealias CGFloat = Foundation.CGFloat
+public typealias CGSize = Foundation.CGSize
+public typealias CGPoint = Foundation.CGPoint
+
+#if os(WASI) || os(Linux)
+    import Glibc
+    public func sqrt(_ x: CGFloat) -> CGFloat { return x.squareRoot() }
+    public func copysign(_ x: CGFloat, _ y: CGFloat) -> CGFloat {
+        let magnitude = x >= 0 ? x : -x
+        return y >= 0 ? magnitude : -magnitude
+    }
+    public func acos(_ x: CGFloat) -> CGFloat {
+        return Glibc.acos(x)
+    }
+
+    public func cos(_ x: CGFloat) -> CGFloat {
+        return Glibc.cos(x)
+    }
+
+    public func sin(_ x: CGFloat) -> CGFloat {
+        return Glibc.sin(x)
+    }
+
+    private let KAPPA: CGFloat = 0.5522847498  // 4 *(sqrt(2) -1)/3
+
+    public struct CGAffineTransform: Equatable {
```

Comment:
Could you consider if we can use `Foundation.AffineTransform`?

Reply:
checking

Reply:
not directly translatable as the props are stored differently in `AffineTransform` under 

```
 public var m11: CGFloat

    public var m12: CGFloat

    public var m21: CGFloat

    public var m22: CGFloat
```

Not going to change for now